cluster/gce/gci/configure-helper.sh

```diff
@@ -1957,6 +1957,7 @@ function start-kube-controller-manager {
   sed -i -e "s@{{pv_recycler_volume}}@${PV_RECYCLER_VOLUME}@g" "${src_file}"
   sed -i -e "s@{{flexvolume_hostpath_mount}}@${FLEXVOLUME_HOSTPATH_MOUNT}@g" "${src_file}"
   sed -i -e "s@{{flexvolume_hostpath}}@${FLEXVOLUME_HOSTPATH_VOLUME}@g" "${src_file}"
+  sed -i -e "s@{{cpurequest}}@${KUBE_CONTROLLER_MANAGER_CPU_REQUEST}@g" "${src_file}"
```

Comment:
This is risky - if someone will call that and not set the vars (which may happen - happy to talk offline about that), the it will crash.
Instead of that you should default it here to appropriate value (and then you don't need defaulting in config_default and config_test).

Reply:
Thank you for the suggestion. I have moved default files to this file.